Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -34,19 +34,6 @@ class MessagingInAppImplementationTest: UnitTest {
         XCTAssertFalse(inAppProviderMock.setProfileIdentifierCalled)
     }
 
-    // MARK: initialize given an existing identifier
-
-    func test_initialize_givenExistingIdentifier_expectGistSetProfileIdentifier() {
-        let givenProfileIdentifiedInSdk = String.random
-
-        profileStoreMock.identifier = givenProfileIdentifiedInSdk
-
-        _ = MessagingInAppImplementation(diGraph: diGraphShared, moduleConfig: moduleConfigDefault)
-
-        XCTAssertTrue(inAppProviderMock.setProfileIdentifierCalled)
-        XCTAssertEqual(inAppProviderMock.setProfileIdentifierReceivedArguments, givenProfileIdentifiedInSdk)
-    }
```

Comment:
Removed, since initialisation is now [not dependent on `profileIdentifier`](https://github.com/customerio/customerio-ios/blob/3614332b4bb13e94a93f41c185bd00692355ed42/Sources/MessagingInApp/MessagingInAppImplementation.swift#L28).

Reply:
Ah, got it. That [FIXME note](https://github.com/customerio/customerio-ios/blob/b850ac74bdf2b48bff78f709996d97c207b3279b/Sources/MessagingInApp/MessagingInAppImplementation.swift#L27C17-L27C17) had me thinking we were we were ditching the profileIdentifier requirement for initialisation.